htp/htp_util.c

```diff
@@ -2063,34 +2063,34 @@ void fprint_raw_data_ex(FILE *stream, const char *name, const void *_data, size_
     while (offset < len) {
         size_t i;
 
-        sprintf(buf, "%08" PRIx64, (uint64_t) offset);
-        strcat(buf + strlen(buf), "  ");
+        snprintf(buf, sizeof(buf), "%08" PRIx64, (uint64_t) offset);
+        strlcat(buf + strlen(buf), "  ", 2);
```

Comment:
in strlcat usage, you can actually do: "strlcat(buf, "   ", sizeof(buf));" Strlcat will by itself figure out where to append the data and make sure that it stays within the limits of the buffer size as given by sizeof(buf)